STYLEGUIDE.md

```diff
@@ -1,5 +1,12 @@
 # Documentation Style Guide
 
+### active voice
+Use the active voice. See [personal pronouns](#personal-pronouns) for guidance 
+on when it is appropriate to use "you" and "your".
+
+* Bad: The files should be added to the directory by the owner.
+* **_Good_: The owner should add the files to the directory.**
+
```

Comment:
```suggestion
* Bad: To access a network's LDAP services, your computer must first be authenticated
* **_Good_: To access a network's LDAP services, a server that supports the protocol must first authenticate your computer

```

Reply:
@nijikokun how's this?

Reply:
I like it! It's a good example, employing both active voice and a valid use of a personal pronoun (the examples are missing `.` at the end, though)

Reply:
fixed, thank you @hishamhm !